components/Icons/index.tsx

```diff
@@ -35,3 +39,112 @@ export const DownIcon: (props: React.SVGProps<SVGSVGElement>) => React.ReactElem
+      <circle cx="29.5" cy="30.5" r="2.5" fill="#FF4A4A" />
+      <mask id="a" width="21" height="22" x="28" y="10" maskUnits="userSpaceOnUse" style={{ maskType: "alpha" }}>
+        <path fill="#fff" d="M48.8559 10.5098H28.3965v21.1648h20.4594V10.5098Z" />
+        <path fill="#000" fillRule="evenodd" d="m42.909 15.7654-.1443-1.4863c-.0203-.2094.0544-.4169.2037-.5657l1.5918-1.5859c.3766-.3752 1.0186-.2164 1.1752.2908l.3455 1.1184c.0697.2259.2485.4018.476.4685l1.0508.3079c.5146.1508.6797.7956.3003 1.1736l-1.5844 1.5785c-.1467.1463-.3506.2208-.5575.2039l-1.3847-.1131-13.6369 13.5869c-.146.1456-.3486.2202-.5546.2042l-.4031-.0312c-.3471-.027-.6228-.3017-.6498-.6475l-.0314-.4016c-.016-.2052.0589-.4071.2049-.5526L42.909 15.7654Z" clipRule="evenodd" />
+      </mask>
+      <g mask="url(#a)">
+        <path fill="#fff" stroke="#FF4A4A" strokeLinejoin="round" strokeWidth="1.2" d="m47.908 15.4867-1.5844 1.5785c-.1467.1463-.3506.2208-.5575.2039l-1.3847-.1131-13.6369 13.5869M47.908 15.4867 30.1899 30.9471c.206.016.4086-.0586.5546-.2042M47.908 15.4867c.3794-.378.2143-1.0228-.3003-1.1736l.3003 1.1736ZM30.7445 30.7429l.4235.4251.0001-.0001m-.4236-.425.4236.425m0 0 13.4405-13.3913 1.1086.0905h.0001c.3817.0312.7583-.1063 1.0298-.3768 0-.0001 0-.0001.0001-.0001l1.5843-1.5785c.7039-.7014.3954-1.8958-.5551-2.1744l-1.0508-.3079c-.035-.0103-.0613-.037-.0714-.0696l-.0001-.0002-.3454-1.1183c-.2904-.9404-1.4771-1.2313-2.172-.5388l-1.5917 1.5858c-.2765.2754-.4152.6601-.3775 1.0488l.1169 1.2037L28.887 28.8891v.0001c-.2702.2693-.4092.6434-.3795 1.0242v.0001l.0313.4016v.0001c.0502.642.5615 1.1492 1.2016 1.1989l.4031.0312h.0001c.3799.0294.7542-.1081 1.0245-.3774Z" />
+      </g>
+    </svg>
+  );
+}
+
+export const SuccessTokenIcon: (props: React.SVGProps<SVGSVGElement>) => React.ReactElement = (props) => {
+  return (
+    <svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 60" width={60} height={60} {...props}>
+      <g clipPath="url(#clip0_1370_2856)">
+        <path d="M30.2488 60C46.6517 60 59.9488 46.5685 59.9488 30C59.9488 13.4315 46.6517 0 30.2488 0C13.846 0 0.548828 13.4315 0.548828 30C0.548828 46.5685 13.846 60 30.2488 60Z" fill="#FF4A4A" />
+      </g>
+      <path d="M16.3887 43.4004H44.3067" stroke="white" strokeWidth="2.1" strokeLinecap="round" />
+      <path d="M30.3484 38.2C36.4174 38.2 41.3374 33.2304 41.3374 27.1C41.3374 20.9696 36.4174 16 30.3484 16C24.2793 16 19.3594 20.9696 19.3594 27.1C19.3594 33.2304 24.2793 38.2 30.3484 38.2Z" fill="#FF4A4A" stroke="white" strokeWidth="2.1" />
+      <path d="M26.7832 27.2161L29.7724 30.7L34.5052 23.5" stroke="white" strokeWidth="2.1" strokeLinecap="round" strokeLinejoin="round" />
+      <defs>
+        <clipPath id="clip0_1370_2856">
+          <rect width="59.4" height="60" fill="white" transform="translate(0.548828)" />
+        </clipPath>
+      </defs>
+    </svg>
+  );
+}
+
+export const RangeTokenIcon: (props: React.SVGProps<SVGSVGElement>) => React.ReactElement = (props) => {
```

Comment:
Could we use something like https://github.com/gregberge/svgr to do this automatically?

Reply:
I tried to use `svgr` but I encountered some issues with it. Some svgs weren't resized correctly when using styled components (they were clipped instead of scaled) and some icons weren't rendered correctly after svgr processed them (missing lines or other elements).

Reply:
Got it, I had encountered something similar, and it was because the svgs used the same id for clips and gradients, anyway, this approach is perfectly fine.